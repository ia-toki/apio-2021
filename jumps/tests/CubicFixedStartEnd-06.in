209636825ecd2c38299498cd26890ef71dd128c2
200 200
1 2 3 4 157 108 7 8 9 84 11 30 172 14 13 16 120 10 19 20 21 22 23 24 25 26 39 28 29 89 31 32 33 136 100 36 190 118 151 134 41 42 83 44 45 46 47 178 49 50 51 52 158 54 55 64 57 58 48 60 61 70 75 87 65 180 67 68 69 194 71 72 73 74 63 76 140 179 138 80 155 82 127 18 85 101 121 88 12 90 91 92 173 94 95 162 97 98 113 43 193 38 103 164 105 40 147 6 109 96 111 125 130 114 34 56 117 77 119 161 116 122 167 124 106 168 152 128 181 153 142 132 133 79 135 159 129 149 139 102 192 186 59 143 145 146 107 62 196 150 66 93 141 154 17 156 144 53 115 160 131 182 126 15 165 175 123 191 176 137 163 104 99 174 166 169 177 5 78 110 112 185 183 200 170 187 81 35 189 188 171 197 37 148 199 195 27 198 86 184
0 0 199 199
145 145 179 179
178 178 199 199
109 109 131 131
56 56 130 130
190 190 191 191
51 51 137 137
76 76 90 90
0 0 17 17
27 27 48 48
15 15 182 182
165 165 181 181
78 78 83 83
137 137 156 156
0 0 28 28
31 31 75 75
107 107 111 111
84 84 95 95
172 172 175 175
42 42 113 113
145 145 189 189
85 85 194 194
150 150 199 199
182 182 190 190
102 102 108 108
169 169 173 173
1 1 186 186
114 114 182 182
147 147 154 154
61 61 130 130
134 134 158 158
67 67 109 109
107 107 111 111
187 187 194 194
198 198 199 199
86 86 199 199
104 104 150 150
132 132 138 138
97 97 153 153
13 13 129 129
73 73 179 179
176 176 185 185
131 131 182 182
22 22 72 72
2 2 161 161
170 170 189 189
117 117 190 190
46 46 133 133
24 24 157 157
176 176 188 188
22 22 40 40
23 23 60 60
78 78 105 105
63 63 197 197
70 70 113 113
66 66 170 170
164 164 178 178
184 184 197 197
136 136 193 193
164 164 187 187
141 141 175 175
163 163 166 166
19 19 161 161
3 3 124 124
157 157 188 188
158 158 185 185
54 54 169 169
167 167 171 171
121 121 145 145
49 49 195 195
149 149 182 182
36 36 68 68
184 184 187 187
117 117 173 173
168 168 175 175
107 107 150 150
156 156 169 169
109 109 127 127
43 43 52 52
135 135 168 168
80 80 178 178
196 196 197 197
61 61 120 120
31 31 52 52
105 105 181 181
60 60 90 90
148 148 168 168
10 10 60 60
43 43 197 197
189 189 197 197
193 193 195 195
47 47 184 184
108 108 142 142
176 176 193 193
14 14 169 169
83 83 101 101
116 116 118 118
151 151 155 155
174 174 184 184
172 172 185 185
51 51 97 97
94 94 144 144
113 113 141 141
98 98 101 101
106 106 184 184
85 85 131 131
135 135 177 177
144 144 185 185
81 81 199 199
78 78 163 163
71 71 87 87
33 33 136 136
131 131 137 137
114 114 174 174
126 126 169 169
106 106 171 171
133 133 140 140
191 191 198 198
118 118 120 120
138 138 143 143
65 65 122 122
184 184 198 198
5 5 65 65
81 81 97 97
88 88 189 189
41 41 137 137
129 129 142 142
138 138 178 178
46 46 196 196
123 123 190 190
61 61 196 196
102 102 119 119
14 14 61 61
13 13 157 157
162 162 186 186
188 188 195 195
1 1 49 49
180 180 193 193
101 101 163 163
54 54 162 162
117 117 173 173
40 40 130 130
170 170 197 197
127 127 143 143
61 61 189 189
86 86 150 150
121 121 133 133
152 152 159 159
71 71 119 119
169 169 197 197
10 10 186 186
12 12 123 123
82 82 186 186
66 66 164 164
68 68 170 170
89 89 126 126
66 66 182 182
1 1 169 169
71 71 122 122
71 71 74 74
21 21 27 27
83 83 103 103
157 157 190 190
138 138 155 155
178 178 183 183
128 128 129 129
0 0 49 49
54 54 138 138
144 144 178 178
95 95 195 195
18 18 67 67
172 172 180 180
84 84 101 101
112 112 166 166
145 145 176 176
159 159 176 176
188 188 199 199
94 94 140 140
194 194 197 197
39 39 117 117
101 101 109 109
44 44 102 102
87 87 114 114
177 177 185 185
194 194 196 196
14 14 113 113
71 71 95 95
196 196 199 199
135 135 171 171
48 48 183 183
35 35 45 45
33 33 89 89
170 170 191 191
102 102 123 123
8 8 72 72
119 119 161 161
13 13 97 97
164 164 192 192
56 56 162 162
171 171 187 187
