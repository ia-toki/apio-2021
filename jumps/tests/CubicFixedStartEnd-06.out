fc5005cd102fffc504bc1615010ebf88823d65f3
OK
-1
-1
-1
-1
-1
1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
3
-1
-1
-1
-1
-1
-1
-1
-1
2
-1
-1
-1
-1
-1
-1
3
3
1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
1
3
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
3
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
3
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
3
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
5
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
